Services/GEV/Utils/classes/class.gevCourseBuildingBlockUtils.php

```diff
@@ -225,9 +227,9 @@ static public function getAllCourseBuildingBlocksRaw($a_crs_ref_id,$a_request_id
 		
 		return $ret;
 	}
-	
+
 	static public function getAllCourseBuildingBlocks($a_crs_ref_id, $a_request_id = null) {
-		return array_map(function($row) {
+		return array_map(function($row) use ($blank_db) {
```

Comment:
This won't work, there is no `$blank_db` here. Didn't see the `static` on the method last time.